tfhe/src/shortint/server_key/compressed.rs

```diff
@@ -7,14 +7,20 @@ use crate::shortint::parameters::{CarryModulus, CiphertextModulus, MessageModulu
 use crate::shortint::ClientKey;
 use serde::{Deserialize, Serialize};
 
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub enum ShortintCompressedBootstrappingKeyBase {
```

Comment:
Same here, we can probably rename `ShortintCompressedBootstrappingKey` to be consistent with the nmaing change I proposed for parameters 😇

Reply:
will do

Reply:
done